cli/azd/internal/appdetect/java.go

```diff
@@ -54,6 +61,11 @@ func (jd *javaDetector) DetectProject(ctx context.Context, path string, entries
 				Language:      Java,
 				Path:          path,
 				DetectionRule: "Inferred by presence of: pom.xml",
+				Options: map[string]interface{}{
+					"parentPath":            currentRoot.path,
+					"posixMavenWrapperPath": currentWrapperPath[0],
+					"winMavenWrapperPath":   currentWrapperPath[1],
```

Comment:
How about using const to avoid using same string in multiple places?

Reply:
fixed